knowledge-base/examples/grid/date-range-filter/main.jsx

```diff
@@ -55,28 +62,52 @@ class App extends React.Component {
       currentFilters.filters.push({
         field: 'FirstOrderedOn',
         operator: event.operator,
-        value: event.value
+        value: event.value,
       });
     } else {
       currentFilters.filters = [];
       currentFilters.logic = 'and';
       currentFilters.filters.push({
         field: 'FirstOrderedOn',
         operator: event.operator,
-        value: event.value
+        value: event.value,
       });
     }
     this.setState({ filter: currentFilters });
   };
 
-  MyDateFilterCell = props => (
-    <DateRangeFilter
-      {...props}
-      min={this.minDateFilter}
-      max={this.maxDateFilter}
-      onDateFilterChange={this.handleDateFilterChange}
-      onDateFilterClear={this.handleClearDateFilter}
-    />
+  MyDateFilterCell = (props) => (
+    <div className="k-filtercell">
+      <DatePicker
+        format={'MMMM yyyy'}
+        defaultValue={defaultValue}
+        value={props.min}
+        onChange={(e) => {
+          this.handleDateFilterChange({
+            value: e.target.value,
+            operator: 'gt',
+          });
+        }}
+      />
+      <DatePicker
+        format={'MMMM yyyy'}
+        value={props.max}
+        onChange={(e) => {
+          this.handleDateFilterChange({
+            value: e.target.value,
+            operator: 'lt',
+          });
+        }}
+      />
+      <button

```

Comment:
Lets try to use our Buttons with svgIcon set. The one below will probably need the fonts styles.